### PR TITLE
[fix] REM points not showing

### LINKED ISSUE #146: REM control point data takes > 15 mins to load
In potree, datasets containing REM control points don't load the control point overlay for a very long time.

Prior to loading (~ 15 mins!):

After load, it appears correctly as:

The associated .fb completes very early in the process (prior to the point cloud downloads):
`veritas-aptiv-2019-02-11/Data/REM-Lane1-Run1_2019-08-01¶002/3_Assessments/control_point_3_rtk_relative.fb`

After the REM points load, CPU utilization spikes to 100%:

### PR DESCRIPTION
**Pull request by NickRizzo-droid**
- closes #146 -- contains some screenshots
- couple of issues:
    - biggest one- starting time window contained strings instead of Numbers. 
        - The code would convert tend & tstart to numbers when the time window was changed which explains why that phenomenon existed
            - animationEngine.js now explicitly converts inputs to Numbers to ensure this does not occur again 
        - see #149 `tracks don't appear after selected until time window is adjusted (need to refresh it or change both sides of the time-window)`
    - error in getting rtk orient which did not let needed values get set
    - updated remLoader.js to predominantly use `const` instead of `let` (helped diagnose issue)
    - lastly, started with >10 points instead of 2-4 points that show when `start` or time window is adjusted
        - due to initial gps time window (which was different from initial time window) being .5s instead of .05 
        - they have been tied to a single point of truth within paramLoader.js

### CODE REVIEW

File: demo/remLoader.js [hunk 1]
```diff
@@ -161,25 +156,23 @@ async function createControlMeshes(controlPoints, remShaderMaterial, FlatbufferM
...
+	const remShaderMaterial = shaderMaterial.clone();
 	await loadRem(s3, bucket, name, remShaderMaterial, animationEngine, (sphereMeshes) => {
-		let remLayer = new THREE.Group();
-		remLayer.name = "REM Control Points";
-		for (let ii=0, len=sphereMeshes.length; ii<len; ii++) {
-			remLayer.add(sphereMeshes[ii]);
-		}
+		const remLayer = new THREE.Group();
+    remLayer.name = "REM Control Points";
+    sphereMeshes.map(mesh => remLayer.add(mesh))
```
Comment (bcasselsND) [inline demo/remLoader.js:164]: Use `forEach`, rather than `map`, since you don't need to collect the values.

File: demo/remLoader.js [hunk 2]
```diff
@@ -113,35 +113,30 @@ async function parseControlPoints(bytesArray, remShaderMaterial, FlatbufferModul
...
+    const point = controlPoints[ii];
+
+    const vertex = {x: point.pos().x(), y: point.pos().y(), z: point.pos().z()};
+    const radius = 0.25;//point.radius();
+    const timestamp = point.viz(new FlatbufferModule.Flatbuffer.Primitives.HideAndShowAnimation())
+    .timestamp(new FlatbufferModule.Flatbuffer.Primitives.ObjectTimestamp())
+    .value() - animationEngine.tstart;
+    
+    
+    const timestampArray = Array(64).fill(timestamp)
```
Comment (bcasselsND) [inline demo/remLoader.js:135]: It looks like the array is filled with numbers. If so, you could use `new Float64Array(64).fill(timestamp)`. That save space and is a little faster.

File: demo/remLoader.js (hunk 1 above)
Comment (NickRizzo-droid) [reply demo/remLoader.js:164]: Good point. I'll switch that

File: demo/remLoader.js (hunk 2 above)
Comment (NickRizzo-droid) [reply demo/remLoader.js:135]: Oh nice! I did not realize you could specify the type of array. I'll do that

**Comment by NickRizzo-droid** [comment]
> I went to a lot of trouble to use strings, rather than numbers, so I could control the number of significant digits shown. I guess that broke things, so I won't worry about it any more....

I could try keeping the strings and only converting to Numbers when they are actually needed by the `THREE` functions/classes (don't know if that would work)

**Pull request merged**